upcoming_changes/2871.enhancements.rst

```diff
@@ -0,0 +1,3 @@
+* Enhancement of drawing.marker
+  1. Add new marker "arrow" and "ellipse"
```

Comment:
```Suggestion
  1. Add new markers "arrow" and "ellipse"
```

Would also make sense to put all on one line.

For the formatting in the changelog, see https://hyperspy--2871.org.readthedocs.build/en/2871/user_guide/changes.html
Currently, there is a double bullet.

Reply:
Thank you.
revised to 1-line.